src/gui/render.cpp

```diff
@@ -823,6 +823,28 @@ void RENDER_InitShaderSource([[maybe_unused]] Section *sec)
 #endif
 }
 
+void RENDER_Init(Section *sec);
+
+static void ReloadShader(bool)
```

Comment:
Maybe double check that it's not being called twice in row.

Key-mapped functions (I think) are called on key-down (bool==true) and key-up (bool=false) events.  

So that's why you'll see a pattern like:

 ``` c++
static void ReloadShader(const bool pressed) {
    if (!pressed)
        return;
    ...
}
```

(But I could be wrong)

Reply:
Ah yes, that's a mistake; the check should definitely be added. Cheers!

Reply:
Oh, just noticed the function's signature need the named `const bool pressed` added.